backend/src/app.e2e.spec.ts

```diff
@@ -13,11 +15,11 @@ describe("AppController (e2e)", () => {
   beforeAll(async () => {
     const mockAppModule: TestingModule = await Test.createTestingModule({
       imports: [AppModule, Neo4jModule.forRoot({
-        scheme: 'neo4j',
-        host: 'localhost',
+        scheme: "neo4j",
```

Comment:
NEVER use credentials in code.. use environment variables (like in the reale AppModule)